### ISSUE TITLE
Incorrect Status Of Device In Kapua

### ISSUE THREAD

**Issue by LeoNerdoG** [eclipse/kapua#2193]
If user first connects a device with proper credentials (proper topic.context.account-name, username and password fields in Kura) and after that changes the topic.context.account-name parameter so it is no longer valid, device is still shown as "Connected", even if user clicks Refresh on the top toolbar. 
This results in enabled bottom tabs (Packages, Bundles, Configuration, Command, Assets) but their parameters are not accessible (stuck in "Loading" screen).

**Testflow:**
1. Login as kapua-sys or any other user that has proper permissions
2. Connect a live Kura device to Kapua (e.g. device0) so status of the device will be Connected
3. Check if device is properly connected in Devices, check if all tabs work
4. in Kura, click Disconnect and refresh status in Kapua. Device should be disconnected
5. Go to Kura, set anything except right topic.context.account-name and click Connect
6. In Kapua go to Devices and click Refresh several times and see if the device is still Connected.
7. Try to access tabs that are normaly disabled if device is disconnected.
**Kapua reports that device is not connected, although user sees that it is connected and after pressing Refresh button, status does not change.**

**Expected behavior**
If previously connected device now has incorrect topic.context.account-name parameter device should be shown as disconnected in Devices tab or Packages, Bundles, Configuration, Command, Assets tabs should be disabled.

**Screenshot:**
![screenshot 2018-11-20 at 09 49 57](https://user-images.githubusercontent.com/15121999/48761782-c1367580-eca9-11e8-8020-3662dc05a257.png)

**Version of Kapua**
1.1.0-SNAPSHOT

**Type of deployment**
[ ] Local Vagrant deployment
[x] Docker
[ ] Openshift (in its variants) 
[ ] Others

**Main component affected**
[x] Console (in case of console please report info on which browser you encountered the problem)
[ ] REST API
[ ] Message Broker
[ ] - Others

Browsers used: 
- Google Chrome (Version 69.0.3497.100 (Official Build) (64-bit))
- Safari (Version 12.0 (13604.3.5))
- Mozilla Firefox (63.0 (64-bit))
- Opera (Version 55.0.2994.44)

**Comment by Coduz** [comment]
This is caused by  the fact that the connection and the device are ok, but the Device is writing and reading in channels in which it has no permissions to. 

Actually the device is connected, but is not listening on the right namespace. 

I think that the shown status is correct, since the platform has no way to check for the namespace used to subscribe/publish. Maybe we can handle more gracefully the Timeout exception that will occur when trying to perform device management operations

**Comment by LeoNerdoG** [comment]
@Coduz,
Yes, from what you have wrote, I see no other option. We could also modify the existing message a bit or add another one just for this matter.
